packages/design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -196,7 +195,7 @@ function Pagination({
      * Renders all Page components in range (3 pages) to Pagination component.
      */
 
-    pageRange.map((page) => {
+    pageRange.forEach((page) => {
```

Comment:
Since `map` isn't returning anything, `forEach` is the right choice here.

Reply:
Nice, yep, that makes sense 👍